lib/bt/constants.js

```diff
@@ -1,32 +1,14 @@
 const MINUTE = 60 * 1000
-const HOUR = 60 * MINUTE
-const DAY = 24 * HOUR
 
 const MAX_ITEMS_PER_REQUEST = 10000
 const MAX_REQUESTS_PER_MINUTE = 90
```

Comment:
I am getting rate limit error so, we should reduce this value. `50` is working fine on my end. The api doc says `The current rate limit is between 10 and 90 requests per minute, depending on the specific REST API endpoint (i.e. /ticker). If an IP address is rate limited, the IP is blocked for 60 seconds and cannot make any requests during that time.` . And we are running parallelly to request the trade and candle data when user selects `useTrades` option as well. 

Might not be efficient solution, but may be we can add the following as well to avoid such case. We can catch this error and add delay for 60 seconds instead of `THROTTLE` value?